vnext/src-win/Libraries/Text/Text.windows.js

```diff
@@ -297,88 +216,169 @@ const Text: React.AbstractComponent<TextProps, TextForwardRef> =
-          styleProps.borderBottomWidth != null ||
-          styleProps.borderEndWidth != null ||
-          styleProps.borderLeftWidth != null ||
-          styleProps.borderRightWidth != null ||
-          styleProps.borderStartWidth != null ||
-          styleProps.borderTopWidth != null)
-      ) {
-        return (
-          <View style={styleProps}>
-            <TextAncestor.Provider value={true}>
-              {nativeText}
-            </TextAncestor.Provider>
-          </View>
-        );
-      }
-      // Windows]
+      return (
+        <NativeVirtualText
+          {...restProps}
+          accessibilityLabel={_accessibilityLabel}
+          accessibilityState={_accessibilityState}
+          accessibilityLevel={_accessibilityLevel} // Windows
+          accessibilityPosInSet={_accessibilityPosInSet} // Windows
+          accessibilitySetSize={_accessibilitySetSize} // Windows
+          isHighlighted={false}
+          isPressable={false}
+          nativeID={_nativeID}
+          numberOfLines={_numberOfLines}
+          ref={forwardedRef}
+          selectable={_selectable}
+          selectionColor={_selectionColor}
+          style={_style}
+          disabled={disabled}>
+          {children}
+        </NativeVirtualText>
+      );
+    }
 
-      if (children == null) {
-        return nativeText;
-      }
+    // If the disabled prop and accessibilityState.disabled are out of sync but not both in
+    // falsy states we need to update the accessibilityState object to use the disabled prop.
+    if (
+      _disabled !== _accessibilityStateDisabled &&
+      ((_disabled != null && _disabled !== false) ||
+        (_accessibilityStateDisabled != null &&
+          _accessibilityStateDisabled !== false))
+    ) {
+      _accessibilityState = {..._accessibilityState, disabled: _disabled};
+    }
 
-      // If the children do not contain a JSX element it would not be possible to have a
-      // nested `Text` component so we can skip adding the `TextAncestor` context wrapper
-      // which has a performance overhead. Since we do this for performance reasons we need
-      // to keep the check simple to avoid regressing overall perf. For this reason the
-      // `children.length` constant is set to `3`, this should be a reasonable tradeoff
-      // to capture the majority of `Text` uses but also not make this check too expensive.
-      /*
-      if (Array.isArray(children) && children.length <= 3) {
-        let hasNonTextChild = false;
-        for (let child of children) {
-          if (child != null && typeof child === 'object') {
-            hasNonTextChild = true;
-            break;
-          }
-        }
-        if (!hasNonTextChild) {
-          return nativeText;
+    const _accessible = Platform.select({
+      ios: accessible !== false,
+      android:
+        accessible == null
+          ? onPress != null || onLongPress != null
+          : accessible,
+      default: accessible !== false,
```

Comment:
just curious, is this specific for windows? If so, we should add a comment so it doesn't get removed in future integrations

Reply:
Yes added a comment for future reference.